internal/client/grpc/auth/client.go

```diff
@@ -0,0 +1,28 @@
+package auth
```

Comment:
I am not sure about `grpc` being in `internal` because that will prevent anyone from importing and using our gRPC clients' implementation. Consider using `pkg`.

Reply:
In `internal/clients/grpc`  a new GRPC client connections is create with library `google.golang.org/grpc`

And the GRPC API  of THINGS `github.com/mainflux/mainflux/things/api/auth/grpc` and GRPC API of AUTH `github.com/mainflux/mainflux/auth/api/grpc` passed to the newly created GRPC client connection .   

Anyone can use the THINGS GRPC API `github.com/mainflux/mainflux/things/api/auth/grpc` 
 and AUTH GRPC API `github.com/mainflux/mainflux/auth/api/grpc`  to access them